helmfile/01-applications.yaml

```diff
@@ -235,18 +235,18 @@ releases:
 {{ end }}
 
 
-#Elasticsearch-prometheus-exporter
-- name: elasticsearch-exporter
+# prometheus-elasticsearch-exporter
+- name: prometheus-elasticsearch-exporter
```

Comment:
Could we skip `prometheus-` in the name and label? It is nice and short and the same as before. :slightly_smiling_face:

Reply:
Ususally you want the relase to be the same as the chart name for nice naming, https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-elasticsearch-exporter/templates/_helpers.tpl#L12
Not 100% sure for this chart though.

Reply:
Oh I was not aware of that! Let's keep it then :slightly_smiling_face:

Reply:
I considered that as well, but I wasn't sure which way to go about it. I kind of like including the `prometheus-` just because it makes it more clear what this chart is used for, but I understand the argument for keeping it as it was. I'd be fine with changing it back.